modules/administration-guide/pages/installing-che-on-codeready-containers.adoc

```diff
@@ -10,10 +10,69 @@
 
 :context: installing-{prod-id-short}-on-codeready-containers
 
-This chapter describes the creation of a single-node OpenShift cluster for deploying a {prod-short} instance using CodeReady Containers. To install {prod-short}, use the`{prod-cli}` command-line tool or the OpenShift web console GUI.
+This chapter describes the creation of a single-node OpenShift cluster for deploying a {prod-short} instance using CodeReady Containers.
 
-include::partial$proc_using-chectl-to-install-che-on-codeready-containers.adoc[leveloffset=+1]
+.Prerequisites
 
-include::partial$proc_using-openshift-web-console-to-install-che-on-codeready-containers.adoc[leveloffset=+1]
+* The `{prod-cli}` management tool is available. See xref:installing-the-chectl-management-tool.adoc[].
+* An installation of CodeReady Containers. See link:https://console.redhat.com/openshift/create/local[Installing CodeReady Containers].
+* The path to the user's CodeReady Containers pull secret is known as __<pull-secret-file>__.
+* Available memory: 12 GB or more.
+* Available disk size: 64 GB or more.
 
-:context: {parent-context-of-installing-che-on-codeready-containers}
+.Procedure
+
+. Remove any previous cluster:
++
+----
+$ crc delete
+----
+
+. Configure your host machine for CodeReady Containers:
++
+----
+$ crc setup
+----
+
+. Start the CodeReady Containers virtual machine with at least 12 GB of memory and 64 GB of disk space. Also, specify the path to your pull secret and take note of the password for the `kubeadmin` user displayed at the end of the CodeReady Containers initiation:
++
+[subs="+quotes"]
+----
+$ crc start --memory 12288 --disk-size=64 --pull-secret-file __<pull-secret-file>__
+----
+
+. Enable access to the `oc` command line interface embedded in CodeReady Containers:
++
+----
+$ eval $(crc oc-env)
+----
+
+. Install the {prod-short} instance using {prod-cli}:
++
+[subs="+attributes"]
+----
+$ {prod-cli} server:deploy --platform crc
+----
+
+.Verification steps
+
+. Verify the {prod-short} instance status:
++
+[subs="+attributes,+quotes"]
+----
+$ {prod-cli} server:status
+----
+
+. Navigate to the {prod-short} cluster instance:
++
+[subs="+attributes,+quotes"]
+----
+$ {prod-cli} dashboard:open
+----
+
+
+
+.Additional resources
+
+* link:https://access.redhat.com/documentation/en-us/red_hat_codeready_containers/[Product documentation for Red Hat CodeReady Containers].
+* link:https://github.com/code-ready/crc[CodeReady Containers on GitHub].
```

Comment:
Avoid linking to GitHub. Instead, try to incorporate the required information directly into our documentation so that it goes through the same QE verification process as the rest of the content. Exceptions to this rule must be okayed by the relevant Product Experience team rep (for OpenShift, that would be Eric Rich).

Reply:
```suggestion
```